.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: "[Issue]"
+labels: 'T: defect'
+assignees: ''
+
+---
+
+**Bug Description**
+<!--A clear and concise description of the problem.-->
+
+**Steps to Reproduce**
+<!--Steps to reproduce the behavior (input file, or modifications to an existing input file, etc.)-->
+
+**Impact**
+<!--Does this prevent you from getting your work done, or is it more of an annoyance?-->
```

Comment:
I think this description might encourage uninformative responses like "This bug prevents me from getting work done", which I'm guessing is not what is wanted. I think we want some measure of the severity of the bug, and some more information about why it's that level of severity. I'll suggest:
```
How does the bug impact your work? What is the severity?
```
Also, I think if people sometimes just read the header "Impact", they might think "Fix Impact" rather than "Bug Impact", so maybe call it "Bug Impact". However, they *should* read the description, so maybe this isn't that important.